howfairis/workarounds/github_caching.py

```diff
@@ -0,0 +1,21 @@
+from datetime import datetime
+from datetime import timedelta
+import requests
+from dateutil import tz
+
+
+def github_caching_check(checker):
+    try:
+        response = requests.get(checker.repo.api)
+        date_created_string = response.json().get("created_at")
```

Comment:
Hi Erik, can you point me to where the meaning of this key is described?

Reply:
Sorry, I do not understand the word "key" in this context. Do you mean "created at"? I got that from an example output the Github API generated.